search/create_datastore_and_search.ipynb

```diff
@@ -0,0 +1,3120 @@
+{
```

Comment:
<p>Why is the function for <code>import_single_document()</code> </p><p> </p><p>I don't think you need to include the Cloud Storage library, you can just create the <code>ImportDocumentsRequest</code> with multiple uris or a pattern</p><p> </p><p><a href="https://cloud.google.com/python/docs/reference/discoveryengine/latest/google.cloud.discoveryengine_v1alpha.types.ImportDocumentsRequest" target="_blank">https://cloud.google.com/python/docs/reference/discoveryengine/latest/google.cloud.discoveryengine_v1alpha.types.ImportDocumentsRequest</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/271/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>